docs/docs/integrations/chat/google_vertex_ai_palm.ipynb

```diff
@@ -205,12 +200,14 @@
    "source": [
     "## Asynchronous calls\n",
     "\n",
-    "We can make asynchronous calls via the Runnables [Async Interface](/docs/expression_language/interface)"
+    "We can make asynchronous calls via the Runnables [Async Interface](/docs/expression_language/interface).\n",
+    "\n",
+    "Note: System message is not yet supported by Gemini models."
```

Comment:
we released a [partial] support for a system message yesterday, could you update this piece, please?

Reply:
I'm not sure it was updated, am I missing anything?

Reply:
it was added by someone else, already in master, that's why it's not shown here in diff
https://github.com/manokhina/langchain/blob/master/docs/docs/integrations/chat/google_vertex_ai_palm.ipynb